packages/core-api/src/versions/2/delegates/methods.ts

```diff
@@ -17,6 +18,18 @@ const index = async request => {
     return toPagination(request, delegates, "delegate");
 };
 
+const active = async request => {
+    const height = this.blockchain.getLastHeight();
+
+    const delegates = await databaseService.delegates.getActiveAtHeight(request.query.height || height);
```

Comment:
```suggestion
    const delegates = await databaseService.delegates.getActiveAtHeight(request.query.height || this.blockchain.getLastHeight());
```

Otherwise the `this.blockchain.getLastHeight()` always get called even if a height was provided.